app/models/user.rb

```diff
@@ -82,6 +82,7 @@ class User
   # Validations
   # email presence validated through Devise
   validates :name, presence: true
+  validates :role, presence: true
```

Comment:
Would it be helpful to declare a `default: :cm` on this model's `role` field as well? Do you think that would be a useful clue on what role was supposed to be if you were looking at the user model for the first time?